main.qml

```diff
@@ -1692,6 +1692,9 @@ ApplicationWindow {
             informationPopup.open();
         }
         onRejectedNewPassword: {}
+        Keys.enabled: !passwordDialog.visible
```

Comment:
```suggestion
        Keys.enabled: !passwordDialog.visible && informationPopup.visible
```

I'm not sure how this Keys.enabled things works honestly, but does it make sense to add this? So that .close() can't be called if the popup isn't open.

Reply:
@selsta I was trying to figure out how it works as well. This makes more sense to me. Let me update and retest.